### PR TITLE
Perturbation space signature implementation

### PR DESCRIPTION
**Pull request by Koncopd**
**PR Checklist**

-   [x] Referenced issue is linked
-   [ ] If you've fixed a bug or added code that should be tested, add tests!
-   [ ] Documentation in `docs` is updated

**Description of changes**

**Technical details**

**Additional context**

### CODE REVIEW

File: pertpy/tools/_pert_sign.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    the most similar mRNA expression profiles. The perturbation signature is calculated by subtracting
+    the averaged mRNA expression profile of the control neighbors from the mRNA expression profile
+    of each cell.
+
+    Args:
+        adata: The annotated data object.
+        pert_key: The column  of `.obs` with perturbation categories, should also contain `control`.
+        control: Control category from the `pert_key` column.
+        split_by: Provide the column `.obs` if multiple biological replicates exist to calculate
+            the perturbation signature for every replicate separately.
```
Comment (Zethson) [inline pertpy/tools/_pert_sign.py:35]: I think that the words in the new line have to match the indentation in the line above or Sphinx will complain.

-> 

```
  split_by: Provide the column `.obs` if multiple biological replicates exist to calculate
                the perturbation signature for every replicate separately.
```

**Review by Zethson — COMMENTED** [review]
Super minor comment. Great work! Much appreciated.

File: pertpy/tools/_pert_sign.py (hunk 1 above)
Comment (Koncopd) [reply pertpy/tools/_pert_sign.py:35]: Hm, I followed the examples from [the link](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

Comment (Zethson) [reply pertpy/tools/_pert_sign.py:35]: All right. My memory might be off. I'll happily test it as soon as you've added it to the usage.md. I can also do it later myself, no problem.

**Comment by Koncopd** [comment]
@Zethson can i merge?

**Comment by Zethson** [comment]
I'll take a look at it later today/tomorrow and then merge. No worries!

**Comment by Zethson** [comment]
Thank you!

**Pull request merged**